envi/qt/memcanvas.py

```diff
@@ -259,8 +259,23 @@ def _add_raw(self, text, cb=None):
         self._appendInside(text, cb)
 
     def addText(self, text, tag=None, cb=None):
-        text = html.escape(text)
-        #text = text.replace('\n', '<br>')
+        text = html.escape(text)\
```

Comment:
This seems to work the same and is a lot cleaner?
`text = html.escape(text).encode('unicode_escape').decode('utf-8')`

But you might wanna double check me on what side effects that might have

Reply:
lemme check this out.  i was hoping you'd pipe up with some brilliance about how to do this better :)